### DISCUSSION TITLE
Degradation acceleration according to c-rate

### DISCUSSION THREAD

**Discussion by YouHyin** [pybamm-team/PyBaMM#3233] [Q&A]
Hi, I'm currently studying the effect of degradation mechanisms with c-rate. In pybamm I'm studying sei growth and lithium plating and it's hard to find material where this is directly in the equation (acceleration of degradation with c-rate) Can you recommend materials within pybamm, maybe could I look at pybamm/models/submodels/interface/kinetic(https://github.com/pybamm-team/PyBaMM/tree/develop/pybamm/models/submodels/interface/kinetics)?

**Comment by brosaplanella** [comment]
The kinetics do not depend directly (or only) on C-rate, but through the other model variables. I think the best approach here is to run the various models for SEI and plating and compare the results.
